scripts/firedrake-install

```diff
@@ -1221,10 +1227,11 @@ if mode == "install":
         packages.remove("petsc4py")
         packages.remove("firedrake")
         v = sys.version_info
-        easyinstall = file("../lib/python%d.%d/site-packages/easy-install.pth" % (v.major, v.minor), "r").readlines()
-        new_packages = [os.getcwd() + "/" + p + "\n" for p in packages]
-        file("../lib/python%d.%d/site-packages/easy-install.pth" % (v.major, v.minor), "w").writelines(
-            easyinstall[:1] + new_packages + easyinstall[1:])
+        if v.major <= 2:
+            easyinstall = open("../lib/python%d.%d/site-packages/easy-install.pth" % (v.major, v.minor), "r").readlines()
+            new_packages = [os.getcwd() + "/" + p + "\n" for p in packages]
+            open("../lib/python%d.%d/site-packages/easy-install.pth" % (v.major, v.minor), "w").writelines(
+                easyinstall[:1] + new_packages + easyinstall[1:])
```

Comment:
Oh, the original bug... I don't remember, this code snippet has been laying around in this file for a while.